index.rst

```diff
@@ -46,12 +46,77 @@
 
 .. note::
 
-   **This technote is not yet published.**
+   This proposal outlines how observing tasks will be flowed from the planning stage through execution,
+   then linked to data analysis technotes and/or notebooks.
+
+   Similar to what is done in the LVV project,
+   using Jira as a central workflow management tool is key to linking the pieces together, monitoring the process,
+   and making it visible to all the stakeholders.
+
+Motivation for this proposal
+============================
+
+    * Night plans need to point to tasks with procedures rather than having the task description in the plan.
+    * We want central repository for all information related to a task executed at night.
+    * We want to have a way to know how and when the tasks were run and how the data was processed.
+    * Data analysis and results from the data need to be easily related to when the data was taken.
+    * Search and find tasks and their results.
+
+Steps
+=====
+
+    * Run planning
+        * Which tasks/test are we running during a observation run?
+        * Night plans need to point to discrete tasks that are linked to where the procedure and artifacts are archived.
```

Comment:
What do think? Would it be interesting to have some sort of simple engineering task proposal platform? It'd be to mainly make sure all the information is contained and properly described when creating night notebooks or tasks.